cmd/legacy/heartbeat/heartbeat.go

```diff
@@ -42,11 +42,19 @@ func Run(v *viper.Viper) (int, error) {
 			)
 		}
 
+		var errbadRequest api.ErrBadRequest
+		if errors.As(err, &errbadRequest) {
+			return exitcode.ErrDefault, fmt.Errorf(
+				"failed to send heartbeat(s) due to api error: %s",
```

Comment:
will this result in a log line with repeated text? `failed to send heartbeat(s) due to api error: failed to send heartbeat(s) due to api error: ...`